test/units/modules/network/nxos/test_nxos_vlan.py

```diff
@@ -64,11 +65,12 @@ def load_from_file(*args, **kwargs):
 
         self.run_commands.side_effect = load_from_file
         self.load_config.return_value = None
+        self.get_config.return_value = load_fixture('nxos_vlan', 'config.cfg')
 
     def test_nxos_vlan_range(self):
         set_module_args(dict(vlan_range='6-10'))
         result = self.execute_module(changed=True)
```

Comment:
Maybe use `self.execute_module(changed=True, commands=['vlan 6', 'vlan 7', 'vlan 8', 'vlan 9', 'vlan 10'])`? That automatically handles ensuring `commands` retains sort order. (and can add `sort=False` if you want to use it for the others and don't need sorting)